lektor/imagetools.py

```diff
@@ -165,6 +165,15 @@ def iso(self):
     @property
     def created_at(self):
         try:
+            print(self._mapping)
```

Comment:
This pull request can't be merged with debugging statements like this still in the code.